client-admin/Dockerfile

```diff
@@ -1,4 +1,6 @@
 # Gulp v3 stops us from upgrading beyond Node v11
+FROM polis_config AS config
```

Comment:
Should be able to use the `image:tag` format directly in the COPY, rather than adding a no-op stage :) https://docs.docker.com/develop/develop-images/multistage-build/#use-an-external-image-as-a-stage

Reply:
```suggestion
```